### PR TITLE
Add a README.md for Windows

### PR DESCRIPTION
**Pull request by nibanks**
This adds an initial readme file for OpenThread on Windows. It gives a basic architecture overview, build instructions and installation instructions for the driver. You can see it on my branch here:

https://github.com/nibanks/openthread/blob/pr/win-readme/examples/drivers/windows/README.md

### CODE REVIEW

File: examples/drivers/windows/README.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+### otLwf.sys ###
+
+This is where most of the real logic lives. `otLwf.sys` is an [NDIS][ndis] Light Weight Filter ([LWF][lwf]) driver.
+It plugs into the networking stack, binding to a protocol driver (TCPIP) at the top, and an NDIS [Miniport][miniport]
+at the bottom. It's job is to take IPv6 packets from TCPIP and pass the necessary data down to the Miniport
+in order to send the packets out over the network.
+
+`otLwf.sys` supports operating in two modes: Full Stack and Tunnel. Full Stack mode is where OpenThread is 
+running on the host (in Windows) and a simple radio device is connected externally. Tunnel mode is where 
+OpenThread is running on the external device and Windows is meerly a pass through for commands and packets.
```
Comment (jwhui) [inline examples/drivers/windows/README.md]: `meerly` -> `merely`

File: examples/drivers/windows/README.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+This is a 'simple' device, exposing a mostly PHY layer interface (with a few MAC layer constructs).
+
+### OpenThread (NCP) Device ###
+
+This is a 'full solution' device, that exposes the entire OpenThread interface via NCP.
+
+## Build ##
+
+Open the Visual Studio Solution `openthread.sln` under `etc\visual-studio` and then pick the
+configuration and platform you wish to build for. For instance, `Release` and `x64`. build
```
Comment (jwhui) [inline examples/drivers/windows/README.md]: comma instead of `.` after `x64`?

Comment (nibanks) [inline examples/drivers/windows/README.md]: 'build' should have been capitalized.

File: examples/drivers/windows/README.md (hunk 1 above)
Comment (nibanks) [inline examples/drivers/windows/README.md]: Fixing this...

**Comment by beriberikix** [comment]
Minor nit - in the diagram under the external session it says "Openthread (NCP) Device." It should be OpenThread (capital T.)

**Comment by nibanks** [comment]
Next time I update the docs, I will be sure to fix this.

**Pull request merged**